frontend/packages/kubevirt-plugin/integration-tests/tests/vm.snapshots.scenario.ts

```diff
@@ -0,0 +1,59 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import {
+  createResource,
+  click,
+  addLeakableResource,
+  removeLeakedResources,
+} from '@console/shared/src/test-utils/utils';
+import * as editVMSnapshotsView from '../views/vm.snapshots.view';
+import { saveButton } from '../views/kubevirtUIResource.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/constants/common';
+import { VirtualMachine } from './models/virtualMachine';
+import { getVMManifest } from './mocks/mocks';
+import { getRandStr } from './utils/utils';
+import { READY, DELETE } from '../../src/utils/strings';
+import { listViewAction } from '@console/shared/src/test-utils/actions.view';
+
+describe('KubeVirt VM Snapshots', () => {
+  const leakedResources = new Set<string>();
+  const testVM = getVMManifest('Container', testName, `snapshotresourcevm-${getRandStr(5)}`);
+  const vm = new VirtualMachine(testVM.metadata);
+
+  beforeAll(async () => {
+    createResource(testVM);
+    addLeakableResource(leakedResources, testVM);
+  });
+
+  afterAll(() => {
+    removeLeakedResources(leakedResources);
+  });
+
+  it(
+    'ID() create, restore and delete a snapshot',
```

Comment:
@yaacov do we have test `ID` for this PR?

Reply:
It's a question for @rhrazdil or @gouyang.

@rhrazdil , @gouyang do you know how to get/create a test ID ?

Reply:
It needs to create the test case on polarion, if you can't do it in polarion, then I can help creating it.

Reply:
@gouyang I'm not sure how to do it.. we need cases for CDI Upload, and Snapshots

Reply:
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-4717.

Please use `ID(CNV-4717)`.